.npmrc

```diff
@@ -0,0 +1 @@
+loglevel=silent
```

Comment:
This suppresses the useless npm noise when lint (or any other script) fails.

Reply:
Is there any danger in doing this? Like are there any benefits to keeping the NPM noise?

Reply:
In my experience, no. It's a lot of words telling you that something has failed but it's probably not npm's fault, an inverted "npm ERR!" nine times to let you know it's really important, and a pointer to an npm log file that tells you about npm but nothing about the script that failed. And all of this pushes the useful output from the script up and away from your eyes.

I suppose that if there were actually a problem with npm itself, the noise might lead you to look in the log which could help you figure that out, but in the 6 years I've been doing JavaScript development with loglevel=silent, that hasn't ever happened to me.